controller_manager/test/test_spawner_unspawner.cpp

```diff
@@ -83,11 +83,6 @@ TEST_F(TestLoadController, spawner_with_no_arguments_errors)
   EXPECT_NE(call_spawner(""), 0) << "Missing mandatory arguments";
 }
 
-TEST_F(TestLoadController, spawner_without_manager_errors)
-{
-  EXPECT_NE(call_spawner("ctrl_1"), 0) << "Wrong controller manager name";
-}
-
```

Comment:
If I remember correctly, that got removed  since waiting forever made that test not really applicable anymore.

Reply:
Not sure... perhaps a rebase issue. Let's add it back and see!

Reply:
CI seems happy with it :-)

Reply:
Awesome!

Reply:
I just checked. Seems like the CI didn't run as there are conflicts

Reply:
Ah, I only looked at the latest checks result...

Going through the code once more makes it clear that this test will never succeed. It runs forever, since we don't have a timeout specified.

```
9: [INFO 2024-08-11 20:20:02.566] [test_controller_manager]: Received robot description from topic.                                                                                                                                         
9: [WARN 2024-08-11 20:20:02.566] [test_controller_manager]: ResourceManager has already loaded a urdf. Ignoring attempt to reload a robot description.                                                                                     
[Processing: controller_manager]                                                                                                                                                                                                            
[Processing: controller_manager]                                                                                                                                                                                                            
[Processing: controller_manager]                                                                                                                                                                                                            
[Processing: controller_manager]                                                                                                                                                                                                            
 9/11 Test  #9: test_spawner_unspawner ..............................***Timeout 120.11 sec
```

However, instead of deleting the test completely, we could rather add a timeout argument:

```c++
TEST_F(TestLoadController, spawner_without_manager_errors_with_given_timeout)
{
  EXPECT_NE(call_spawner("ctrl_1" --controller-manager-timeout 1.0), 0) << "Wrong controller manager name";
}
```

In fact, since @saikishor has added a couple more tests in the meantime, that addition needs to be added at multiple places.

@bmagyar I've pushed a merge with conflicts resolved and  a modified test to my fork at https://github.com/fmauch/ros2_control/tree/fix/spawner-interrupt